src/commands/logs.ts

```diff
@@ -0,0 +1,32 @@
+// Output the logs of the bots
+
+import { CommandInteraction, PermissionFlagsBits, SlashCommandBuilder } from "discord.js";
+import type { Command } from "@/types.js";
+import fs from "node:fs";
+
+const LOG_PATH = "./bot-logs.txt";
+const TEMP_LOG_PATH = "./bot-logs-temp.txt";
+
+export default {
+    data: new SlashCommandBuilder()
+        .setName("logs")
+        .setDescription("Get the next 4000 logs of the bot")
+        .addIntegerOption(option => option.setName("position").setDescription("Beginning position to seek out logs").setRequired(true))
+        .setDefaultMemberPermissions(PermissionFlagsBits.ViewAuditLog),
+    async execute(interaction: CommandInteraction) {
+        // Get position and next 4000 logs of the bot
+        const logs = fs.existsSync(LOG_PATH) ? fs.readFileSync(LOG_PATH, "utf-8").split(/\r?\n/) : [];
```

Comment:
Use the promises version of filesystem functions when you can (here it's an optimal situation to).
Additionally, you should probably respond to the user telling them that the bot wasn't set up with logging if the file doesn't exist.